OpenLIFUTransducerTracker/OpenLIFUTransducerTracker.py

```diff
@@ -1448,6 +1448,7 @@ def onStartPhotoscanGenerationButtonClicked(self):
             self.updatePhotoscanGeneratorProgressBar(new_photoscan_generator_progress_value = 0)
             self.ui.photoscanGenerationStatusMessage.show()
             self.ui.photoscanGenerationStatusMessage.text = ("Generating mesh... this process can take up to 20 minutes.")
+            self.ui.photoscanGenerationStatusMessage.styleSheet = "color:red;"
```

Comment:
the red is much easier to see, thanks
ugly, but we are okay with ugly for now :)